test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
For the record, my example was meant to be a way you could do this today.

>  i'm assuming (requesting!!!!) that marionette's module system is also going away? please? :)

yes